app/js/components/actions/actionsRule/actionsRule.controller.js

```diff
@@ -144,17 +144,16 @@ function ActionsRuleCtrl(
     $scope.plans = MaintenanceService.plans;
 
     $scope.showSystem = function (system) {
-        let systems;
+        const systems = RhaTelemetryActionsService.getClusterAffectedSystems();
 
-        if (typeof InsightsConfig.actionsShowSystem === 'function') {
-            return InsightsConfig.actionsShowSystem(system.toString());
-        }
-
-        systems = RhaTelemetryActionsService.getClusterAffectedSystems();
         if (typeof system === 'string' && systems && systems.hasOwnProperty(system)) {
             system = systems[system];
         }
 
+        if (typeof InsightsConfig.actionsShowSystem === 'function') {
+            return InsightsConfig.actionsShowSystem(system);
```

Comment:
@iphands  I am ok with returning empty here.
I just moved the code block because that was a bug, but did not want to mess with the original logic.
@jharting - I think you added this one - can you confirm its ok to return empty?